net-misc/balance/balance-3.57-r1.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit toolchain-funcs fcaps
+
+DESCRIPTION="TCP Load Balancing Port Forwarder"
+HOMEPAGE="http://www.inlab.de/balance.html"
+SRC_URI="http://www.inlab.de/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE="caps"
+
+DEPEND="caps? ( sys-libs/libcap )"
```

Comment:
`sys-libs/libcap` is not needed here, `balance` does not use anything from this library and `fcaps` inherit sets its dependencies automatically.

Reply:
Indeed, and even extra flag is nor really required, removing